obspy/io/reftek/core.py

```diff
@@ -319,8 +315,21 @@ def to_stream(self, network="", location="", component_codes=None,
                             sample_data = np.require(
                                 packets_['payload'],
                                 requirements=['C_CONTIGUOUS'])
+                            # account for number of samples
+                            sample_bytes = sample_data.shape[1]
+                            sample_data = sample_data.flatten()
+                            bytes_ = int(encoding) // 8
```

Comment:
I think I would prefer an explicit/static..

```
dtype = {'16': np.int16, '32': np.int32}[encoding]
bytes_ = {'16': 2, '32': 4}[encoding]
```

.. at the top. Just being paranoid but these division operators make me slightly nervous